.github/workflows/clean-up-docker.yaml

```diff
@@ -0,0 +1,38 @@
+name: Clean up Docker Images on PR closure
+
+on:
+  workflow_call:
+
+# Permissions needed
+# permissions:
+#   packages: write
+
+jobs:
+  purge-image-storer:
+    name: Delete image from ghcr.io
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete image
+        uses: bots-house/ghcr-delete-image-action@v1.1.0
+        with:
+          owner: ${{ github.repository_owner }}
+          name: ${{ github.event.repository.name }}
```

Comment:
let's make name as input parameter with default value to `${{ github.event.repository.name }}` so that the workflow can be used also for multi image repositories.